etc/create_brick.py

```diff
@@ -0,0 +1,36 @@
+import etcd
+import json
+import uuid
+
+
+integration_id = "ab3b125e-4769-4071-a349-e82b380c11f4"
+job_id = str(uuid.uuid4())
+
+payload = {
+    "integration_id": integration_id,
+    "run": "tendrl.gluster.flows.CreateGlusterBrick",
+    "parameters": {
+        "GlusterBrick.brick_dict": {
+            "12.34.56.78:/mnt/tendrl_sample4/b7": {
+                "node_id": "4a47db22-b286-44da-8240-c8256b28d256",
+                "disk": "vdh"
+            },
+            "12.34.56.78:/mnt/tendrl_sample5/b8": {
```

Comment:
Done, have standardized the brick path to "/glusterfs"  user now has to provide only brick name. the brick will be mounted under "/glusterfs"